build/function-package.sh

```diff
@@ -20,4 +20,4 @@ rm -f ./artefacts/dynatrace-gcp-function.zip
 (cd ./src/; zip -r ../artefacts/dynatrace-gcp-function.zip ./ -x '*__pycache__*')
 sed -i "s/^GCP_FUNCTION_RELEASE_VERSION=.*/GCP_FUNCTION_RELEASE_VERSION='$TRAVIS_TAG'/" scripts/setup.sh
 zip -r ./artefacts/function-deployment-package.zip ./ext_tools/
-zip -j ./artefacts/function-deployment-package.zip ./scripts/lib.sh ./scripts/setup.sh ./scripts/uninstall.sh ./activation-config.yaml
```

Comment:
Can we change the typo in the path here from `artefacts` to `artifacts` or will it cause too much collateral damage?

Reply:
I like this expression "collateral damage". Never know what you will break 😆

Reply:
@wojtasskorcz where do you see typo? https://dictionary.cambridge.org/pl/dictionary/english/artefact

Reply:
"Artefact is the original British English spelling. Artifact is the American English spelling. "

Reply:
so everything is ok :)

Reply:
Oh, didn't know. Never seen "artefact" in the wild :) All good then!